### DISCUSSION TITLE
Log which library was used for token replace

### DISCUSSION THREAD

**Discussion by anaumov85** [qetza/replacetokens-task#52] [Q&A] [answered]
Hi, is there a possibility to log which azure library was used for token replace? That would be very useful for case when we have a variable with the same name in multiple libraries linked to pipeline, as well as when we have a lot of libraries used. In v5 version of task I don't see such log - only which value was used for replace.

**Comment by qetza** [comment, marked as answer]
Hi @anaumov85,
This is not possible, when the task is executed it has no information on the variables they are just set as environment variables by the agent before running the task.
